core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -949,6 +949,85 @@ public void modeInUnsupportedFormat() {
         exception.getMessage());
   }
 
+  private void testWeek_Of_Year(String date, int mode, int expectedResult) {
```

Comment:
Mixture of `camelCase` and `snake_case` for function names. Looks like the rest of the file uses `camelCase`. Would be good to conform.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/175/commits/0e612f3681c396d23d395f6c49fb79631357ad01